src/components/Members.tsx

```diff
@@ -17,29 +18,48 @@ import { IMemberQueryOptions as FilterOptions } from "@daostack/client";
 interface RequiredProps
   extends ComponentListProps<Entity, Data, FilterOptions> {
   allDAOs?: boolean;
+  inferred?: boolean;
 }
 
 interface ArcInferredProps {
-  arcConfig: ProtocolConfig | undefined;
+  arcConfig?: ProtocolConfig | undefined;
 }
 
 interface DAOInferredProps {
-  dao: InferEntity | undefined;
+  dao?: InferEntity | undefined;
 }
 
-type ArcProps = RequiredProps & ArcInferredProps;
-type DAOProps = RequiredProps & DAOInferredProps;
+type ArcProps = RequiredProps & ArcInferredProps & DAOInferredProps;
 
 class ArcMembers extends ComponentList<ArcProps, Component> {
   createObservableEntities(): Observable<Entity[]> {
-    const { arcConfig, filter } = this.props;
-    if (!arcConfig) {
-      throw Error(
-        "Arc Config Missing: Please provide this field as a prop, or use the inference component."
-      );
+    const { arcConfig, filter, dao, inferred } = this.props;
+    if (inferred) {
+      if (!dao) {
+        throw Error(
+          "DAO Entity Missing: Please provide this field as a prop, or use the inference component."
+        );
+      }
+      const daoFilter: FilterOptions = filter ? filter : { where: {} };
+      if (!daoFilter.where) {
+        daoFilter.where = {};
+      }
+      daoFilter.where.dao = dao.id;
+      try {
```

Comment:
This try block has been implemented to add an extra validation - something the if dao props is not received it is `{}` so it passes the first validation (`!dao`)

Lmk if this makes sense to you @dOrgJelli

Reply:
This is handled by the above `if (!dao)`, and we're now initializing the contexts with `undefined` so this if condition should succeed and handle the error you're talking about.